config/settings.yaml.example

```diff
@@ -71,6 +71,24 @@
 #  :dynflow:
 #    :enabled: true
 
+# before enabling any telemetry install foreman-telemetry package (telemetry bundler group)
+:telemetry:
+  # prefix for all metrics
+  :prefix: 'fm_rails'
+  # prometheus ruby client library currently does not supprt multi-process web servers
+  :prometheus:
+    :enabled: false
+  :statsd:
+    :enabled: false
+    # IP and port (do not use DNS)
+    :host: '127.0.0.1:8125'
+    # one of 'statsd', 'statsite' or 'datadog'
+    :protocol: 'statsd'
+  :rails:
+    :enabled: false
```

Comment:
If this is false by default, what metrics are collected by default? Just ruby?

Reply:
This is actually destination, I will rename this to `rails_log` to have this less confusing.